seers/detect.py

```diff
@@ -0,0 +1,74 @@
+from imutils.video import VideoStream
+from imutils.video import FPS
+import numpy as np
+import argparse
+import imutils
+import time
+import cv2
+# Code taken from https://www.pyimagesearch.com/
+
+# construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-p", "--prototxt", required=True,
+	help="path to Caffe 'deploy' prototxt file")
+ap.add_argument("-m", "--model", required=True,
+	help="path to Caffe pre-trained model")
+ap.add_argument("-c", "--confidence", type=float, default=0.2,
+	help="minimum probability to filter weak detections")
+args = vars(ap.parse_args())
+
+# initialize the list of class labels MobileNet SSD was trained to
+# detect, then generate a set of bounding box colors for each class
+CLASSES = ["background", "aeroplane", "bicycle", "bird", "boat",
```

Comment:
Don't we just want people?

Reply:
This is just an instantiation of the things that CAN be detected in the model that we are using.  We will be checking all the objects detected to make sure they are "person" objects

Reply:
Or is meant to detect everything and then increment only when people are seen  as shown on line 69?

Reply:
Yep thats what is going on!